dist/tokens/eth/tokens-eth.json

```diff
@@ -6499,7 +6499,7 @@
     "symbol": "DUBI",
     "name": "DUBI",
     "type": "ERC20",
-    "address": "0x9c6Fa42209169bCeA032e401188a6fc3e9C9f59c",
+    "address": "0xEd7fEA78C393cF7B17B152A8c2D0CD97aC31790B",
```

Comment:
Please update the file `src/tokens/eth/0x9c6Fa42209169bCeA032e401188a6fc3e9C9f59c .json` by renaming it to the new address and updating the data inside of it.

Reply:
done!